### DISCUSSION TITLE
微信3.9.6.33版本失效了

### DISCUSSION THREAD

**Discussion by LiHangzls** [huiyadanli/RevokeMsgPatcher#613] [Q&A] [answered]
我今天早上更新了微信到3.9.6.33版本，突然发现撤回消息有提示，从手机端看确实没有防撤回。
关闭了qq和微信，管理员身份启动防撤回软件并安装补丁，再打开微信还是无法生效。
重复多次也不行，请大佬确认一下了，谢谢。/抱拳

**Comment by huiyadanli** [comment]
![image](https://github.com/huiyadanli/RevokeMsgPatcher/assets/15783049/d2f5e9f0-cc50-493d-a405-86f2a74d3dee)

测试有效

**Reply by LiHangzls** [reply]
我这边的情况是，这个页面没有问题，但是在聊天框撤回消息就真的撤回了，我在手机端撤回，电脑端也显示撤回了。所以如果您的电脑端撤回不显示消息的话，那就是我的问题。我已经降版本了，谢谢回复！

**Reply by huiyadanli** [reply]
我测试手机端撤回，电脑端消息还在。你最好确认下补丁是打在了你运行的微信版本下。

**Reply by LiHangzls** [reply, marked as answer]
我刚才又升级了微信，关闭微信重开补丁和软件，成功了！
但是，因为补丁打开是默认的微信地址，我从来没改过，每次更新都没看，我电脑也只有一个微信版本。刚刚更新也关注了地址这个问题，只能选取版本的文件夹（像你截图的那样），安装的文件夹（上一层）不行，这次成功了。我不知道问题出在哪，打扰了。
谢谢！

**Reply by huiyadanli** [reply]
微信部分情况下升级有bug，会出现多版本共存的情况。一般重装微信可以解决。
